encode_test.go

```diff
@@ -604,6 +696,60 @@ people:
 	}
 }
 
+func TestEncodeWithNestedYAML(t *testing.T) {
+	// Represents objects containing stringified YAML, and special chars
+	tests := []struct {
+		value interface{}
+		// If true, expects a different result between when using forced block or not
```

Comment:
It seems that the description of `block` still remains .

Reply:
Addressed